specification/2.0/Specification.adoc

```diff
@@ -924,77 +1101,106 @@ The following example defines a mesh containing one triangle set primitive:
+| `NORMAL`    | `"VEC3"`        | _float_                     | Normalized XYZ vertex normals
+| `TANGENT`   | `"VEC4"`        | _float_                     | XYZW vertex tangents where the W component is a sign value (-1 or +1) indicating handedness of the tangent basis
+| `TEXCOORD_0`| `"VEC2"`        | _float_ +
+                                  _unsigned byte_ normalized +
+                                  _unsigned short_ normalized | UV texture coordinates for the first set
+| `TEXCOORD_1`| `"VEC2"`        | _float_ +
+                                  _unsigned byte_ normalized +
+                                  _unsigned short_ normalized | UV texture coordinates for the second set
 | `COLOR_0`   | `"VEC3"` +
-                `"VEC4"`        | `5126` (FLOAT) +
-                                  `5121` (UNSIGNED_BYTE) normalized +
-                                  `5123` (UNSIGNED_SHORT) normalized | RGB or RGBA vertex color
-| `JOINTS_0`  | `"VEC4"`        | `5121` (UNSIGNED_BYTE)
-                                  `5123` (UNSIGNED_SHORT)            | See <<skinned-mesh-attributes,Skinned Mesh Attributes>>
-| `WEIGHTS_0` | `"VEC4"`        | `5126` (FLOAT) +
-                                  `5121` (UNSIGNED_BYTE) normalized +
-                                  `5123` (UNSIGNED_SHORT) normalized | See <<skinned-mesh-attributes,Skinned Mesh Attributes>>
+                `"VEC4"`        | _float_ +
+                                  _unsigned byte_ normalized +
+                                  _unsigned short_ normalized | RGB or RGBA vertex color linear multiplier
+| `JOINTS_0`  | `"VEC4"`        | _unsigned byte_
+                                  _unsigned short_            | See <<skinned-mesh-attributes,Skinned Mesh Attributes>>
+| `WEIGHTS_0` | `"VEC4"`        | _float_) +
+                                  _unsigned byte_ normalized +
+                                  _unsigned short_ normalized | See <<skinned-mesh-attributes,Skinned Mesh Attributes>>
 |====
 
-`POSITION` accessor *must* have `min` and `max` properties defined.
+`POSITION` accessor **MUST** have its `min` and `max` properties defined.
 
-`TEXCOORD`, `COLOR`, `JOINTS`, and `WEIGHTS` attribute semantic property names must be of the form `[semantic]_[set_index]`, e.g., `TEXCOORD_0`, `TEXCOORD_1`, `COLOR_0`. Client implementations must support at least two UV texture coordinate sets, one vertex color, and one joints/weights set. Extensions can add additional property names, accessor types, and/or accessor component types.
+`TEXCOORD`, `COLOR`, `JOINTS`, and `WEIGHTS` attribute semantic property names **MUST** be of the form `[semantic]_[set_index]`, e.g., `TEXCOORD_0`, `TEXCOORD_1`, `COLOR_0`. All indices for indexed attribute semantics **MUST** start with 0 and be continuous positive integers: `TEXCOORD_0`, `TEXCOORD_1`, etc. Indices **MUST NOT** use leading zeroes to pad the number of digits (e.g. `TEXCOORD_01`).
```

Comment:
I think this should be "consecutive" integers (not "continuous") - to be confirmed by some English guy!